.NET/Microsoft.Recognizers.Text.DateTime/TasksModeConstants.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+
+using Microsoft.Recognizers.Definitions;
+
+namespace Microsoft.Recognizers.Text.DateTime
+{
+    [SuppressMessage("StyleCop.CSharp.NamingRules", "SA1310: CSharp.Naming : Field names must not contain underscores.", Justification = "Constant names are written in upper case so they can be readily distinguished from camel case variable names.")]
+    public static class TasksModeConstants
+    {
+        public const string SYS_DATETIME_DATE = "date";
```

Comment:
No need to re-declare all these values that are standard. There should only be redeclarations for values that differ from regular and Tasks modes. Like the TOD boundaries.

Reply:
removed datetime type specific constants.